test/ha/activator_test.go

```diff
@@ -142,3 +154,66 @@ func assertSLO(t *testing.T, p test.Prober, slo float64) {
 		t.Error("CheckSLO failed:", err)
 	}
 }
+
+type activatorPod struct {
```

Comment:
nit: Couldn't we live without the type and return a `map[string]string` (name to ip) below and iterate over that?

Reply:
done